dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -5700,4 +5700,85 @@ private WorkflowTimelineItemView toWorkflowTimelineItemView(final WorkflowTimeli
                 wfTimeLine.commentDescription(), wfTimeLine.taskId(), wfTimeLine.type());
     }
 
+    /**
+     * Creates a new workflow comment
+     *
+     * @param request HttpServletRequest
+     * @param workflowSchemeForm WorkflowSchemeForm
+     * @return Response
+     */
+    @POST
+    @Path("/comments/{contentletId}")
```

Comment:
I think this contentletId should be in the form

Reply:
why the contentlet is the owner of the comment generated, the form will be the detail of it
I would guess it make some kinda sense for me

Reply:
If the Contetlet if the owner then should it be "{contentletId}/comments"?